src/java/picard/analysis/GcBiasMetricsCollector.java

```diff
@@ -125,13 +126,12 @@ public void acceptRecord(final GcBiasCollectorArgs args) {
             final SAMRecord rec = args.getRec();
```

Comment:
since we are already here, could you add a @Override to this function signature?